website/docs/how-to/how-to-create-and-manage-user-groups.md

```diff
@@ -0,0 +1,73 @@
+---
+title: How to create and manage user groups
+---
+
+:::info availability User groups are available to Unleash Enterprise users since **Unleash 4.14**. :::
+
+This guide takes you through how use user groups to manage permissions on your projects. User groups allow you to manage large groups of users more easily than assigning roles directly to those users.
+
+## Creating user groups
+
+1. Navigate to groups by using the admin menu (the gear icon) and selecting the groups option.
+
+![A visual representation of the current step: the Unleash Admin UI with the steps highlighted to navigate to groups.](/img/create-ug-step-1.png)
```

Comment:
I'm not sure about the alt text on the images here. Starting every description with "a visual representation of the current step" is probably redundant. I think we should remove that bit, but am happy to discuss this if you think otherwise?

Reply:
Yes, I think that makes sense, I'll clean up the alt text a little here

Reply:
Looks much better, thanks for the suggestion, I've just removed the redundant text